src/server/scripts/BrokenIsles/EyeOfAzshara/boss_wrath_of_azshara.cpp

```diff
@@ -101,26 +118,28 @@ struct boss_wrath_of_azshara : public BossAI
         BossAI::JustDied(killer);
 
         DoCastSelf(SPELL_TIDESTONE_OF_GOLGANNETH, true);
+
+        instance->DoRemoveAurasDueToSpellOnPlayers(SPELL_FROST_RESONANCE);
+        instance->DoRemoveAurasDueToSpellOnPlayers(SPELL_MAGIC_RESONANCE);
     }
 
     void ScheduleTasks() override
     {
         events.ScheduleEvent(SPELL_MASSIVE_DELUGE, 22s);
         events.ScheduleEvent(SPELL_RAGING_STORMS, 5s);
         events.ScheduleEvent(SPELL_TIDAL_WAVE_AT, 10s);
-        events.ScheduleEvent(SPELL_MYSTIC_TORNADO, 40s);
-        events.ScheduleEvent(SPELL_ARCANE_BOMB, 60s);
+        events.ScheduleEvent(SPELL_MYSTIC_TORNADO, 20s);
+        events.ScheduleEvent(SPELL_ARCANE_BOMB, 30s, 40s);
+
+        Difficulty instanceDifficulty = instance->instance->GetDifficultyID();
+        if (instanceDifficulty == DIFFICULTY_HEROIC || instanceDifficulty == DIFFICULTY_MYTHIC)
```

Comment:
instance->IsHeroic() (also true for mythic, because all heroic spell continue to work on mythic)